vanilla-framework/vanilla-dashboard-theme
# Icon library

## Done

- Created the full icon set for dashboard theme, icons will automatically take the current color of their parent. Error, Success and Warning are exempt from this rule

## QA

- Pull down the branch and check for any linting issues
- View icons on the light and dark pages

## Details

Fixes #35 

## Screenshots

[if relevant, include a screenshot]
